site/app/libraries/database/DatabaseQueries.php

```diff
@@ -3763,4 +3763,30 @@ public function getHasSubmission(gradeable\Gradeable $gradeable, Submitter $subm
             [$gradeable->getId(), $submitter->getId(), $submitter->getId()]);
         return $this->course_db->row()['exists'] ?? false;
     }
+
+    /**
+     * Gets a list of emails for all active particpants in a course  
+     */
+    public function getClassEmailList(){
+        $parameters = array();
+        $this->course_db->query('SELECT user_email FROM users WHERE user_group != 4 OR registration_section IS NOT null', $parameters);
+
+        return $this->course_db->rows();
+    }
+    /**
+     * Queues an email to be sent by email job
+     * @param array $email_data
+     * @param string $recipient  
+     */ 
+    public function sendEmail($email_data, $recipient){
```

Comment:
let's call this function `createEmail` or `queueEmail` or something to be more accurate what it does.